### DISCUSSION TITLE
what is the Cross validation setup?

### DISCUSSION THREAD

**Discussion by dskhanirfan** [FETS-AI/Challenge#21] [Q&A] [answered]
what is the Cross validation setup? How is the data ratio within the training data used to evaluate and compute the performance metrics? If the data from one colab is used to evaluate after training the Unet from the other colabs in a select epoch or is there any other scheme?

**Comment by sarthakpati** [comment, marked as answer]
- The data is split in an 80 (train)/20 (validation) setup.
- Yes, the validation data from each collaborator is used to evaluate the trained model after each round.

For questions, I would recommend you post on the discussion board directly.

**Reply by dskhanirfan** [reply]
Can you also confirm that the following 28 patients data out of 369 is missing? 149,248, 249, 252,254,255,256,258,259,262,263,267,268,271,281,284,287,289,292,305,307,314,316,317,318,320,324,335 ?

**Reply by sarthakpati** [reply]
Yup, those are indeed not present, as we do not have institutional data for those cases.
